src/apps/components/run.js

```diff
@@ -83,21 +83,12 @@ class RunApp extends React.Component {
     render() {
         return (
             <div className="app-main">
-                <div className="main-view">
-                    <JuttleViewer bundle={this.props.bundle} />
+                <RunHeader {...this.props} onRunClick={this.handleRunClick} ref="runHeader" />
```

Comment:
on second thought... I'd rather not use the `connect` because `RunHeader` is actually a child of `Run` and we do end up using all but one prop from `Run`. Being that they're tightly coupled (and not componenty) I don't see why we can't use `...props`.